tests/files/arxml/system-4.2.arxml

```diff
@@ -1420,6 +1420,7 @@
               <SHORT-NAME>Signal2_SM</SHORT-NAME>
               <DESC>
                 <L-2 L="FOR-ALL">Signal comment! (Sign+Magnitude)</L-2>
+                <L-2 L="FOR-ALL" />
```

Comment:
probably the language should be changed to e.g. `L="EO"`, here, so that "FOR-ALL" does not appear twice. That said, I guess that in autosar duplicate definitions for the same language probably result in undefined behavior...

Reply:
Better? 😄

Reply:
yes :)